packages/aragon-cli/src/cli.js

```diff
@@ -74,63 +73,21 @@ cmd.option('use-frame', {
   default: false,
 })
 
-// network coerce is called multiple times, only warn once
-let warnedDeprecatedNetwork = false
-
-// Ethereum
-cmd.option('network', {
-  description:
-    '(deprecated) The network in your truffle.js that you want to use. Deprecated in favor of `--environment`',
-  coerce: network => {
-    if (warnedDeprecatedNetwork) {
-      return network
-    }
-    warnedDeprecatedNetwork = true
-    reporter.info(
-      'Use of `--network` is deprecated and has been replaced with `--environment`. You may need to update your arapp.json'
-    )
-  },
-})
-
 cmd.option('environment', {
   description: 'The environment in your arapp.json that you want to use',
   // default: 'default'
 })
 
 // APM
-cmd.option('apm.ens-registry', {
```

Comment:
Any particular reason for not having `ens-registry` be configurable in the global options?